### ISSUE TITLE
Shard Tests At The Describe Level So I Can Keep Tests in the Same File

### ISSUE THREAD

**Issue by loganrenz** [angular/protractor#3532]
I have a series of tests that I would like to run in parallel, but I would like to keep them in the same file. I would like to be able to set something like shardTestFiles at the describe level. 

```
describe("Tests for viewing incidents",function(){
	it("test 1",function(){})
	it("test 2",function(){})
});

describe("Tests for creating incidents",function(){
	it("test 3",function(){})
	it("test 4",function(){})
})
```

I would like to be able to run these two "suites" in parallel.

**Comment by juliemr** [comment]
We would have to implement this at the test framework level, which is out of scope for the project. If you're having trouble splitting into two separate files, remember that you can have a helper file and share it with simple `require('helper.js')` node style.
